lib/iris/tests/__init__.py

```diff
@@ -141,6 +141,73 @@ def main():
         unittest.main()
 
 
+def _assert_masked_array(assertion, a, b, strict, **kwargs):
```

Comment:
Pulled these off the test class so I could use them in `pytest` style tests.  Their implementation is unchanged.